src/app/tabs/newAdmin/addresses/operations.js

```diff
@@ -97,21 +102,35 @@ const setMultiChainAddress = (domain, chainId, address, isNew) => async (dispatc
         return dispatch(errorSetChainAddress(chainName, error.message));
       }
 
-      const transactionConfirmed = () => () => {
-        dispatch(receiveSetChainAddress(
-          chainId, getChainNameById(chainId), address, result, isNew,
+      const transactionConfirmed = listenerParams => (listenerDispatch) => {
+        listenerDispatch(receiveSetChainAddress(
```

Comment:
I want to address this. The `listenerParams.var` seems like a smell. I want to use deconstruction here, however, the variable names (chained, address, resultTx, isNew) are already defined in the upper scope. And unless I am missing something, with deconstruction, the variable names need to match the inner names.

Reply:
This should have been reference starting on line 107.

Reply:
> And unless I am missing something, with deconstruction, the variable names need to match the inner names.

This looks good to me. Sometimes a pattern can fail :)

Reply:
Thanks for taking a look at that! Glad this is good even though it looks fishy!